templates/zerver/help/restore-the-last-unsent-message.md

```diff
@@ -1,12 +1,12 @@
 # Restore the last unsent message
 
-![Restore draft](/static/images/help/restore-draft.png)
+If you accidentally exit the compose box while you are composing a message,
+you can restore your previous unsent message.
 
-If you accidentally delete your message by clicking the **x**
-(<i class="icon-vector-remove"></i>) icon in the upper right corner of
-your messaging box, you can restore your previous unsent message by
-clicking a blue link labeled **Restore draft**, which appears between
-the **A** (<i class="icon-vector-font"></i>) and **eye**
-(<i class="icon-vector-eye-open"></i>) icons.
-
-**[Go to "Table of contents"](/help/#using-zulip)**
+1. Compose a [new message or reply](/help/reply-to-a-message).
```

Comment:
I think this should be `open the compose box`, since restoring the draft will remove any messages composed.